draft-ietf-teep-architecture.md

```diff
@@ -735,11 +735,12 @@ Note that personalization data is not included in the table above.
 The use of personalization data is dependent on how TAs are used 
 and what their security requirements are. 
 
-TEEP requests from a TAM to a TEEP Agent can be encrypted with the
-TEE public key (to provide confidentiality), and are then signed with the TAM
+TEEP requests from a TAM to a TEEP Agent can be signed with the TAM
 private key (for authentication and integrity protection).
-Conversely, TEEP responses from a TEEP Agent to a TAM can be encrypted
-with the TAM public key, and are then signed with the TEE private key.  
+Conversely, TEEP responses from a TEEP Agent to a TAM can be 
```

Comment:
```suggestion
Conversely, a TEEP response from a TEEP Agent to a TAM can be encrypted with a data key that is transported with the TAM public key, and is
```

Reply:
We are not providing encryption capabilities for the communication from the TEEP Agent to the TAM.

Reply:
It was the original design that TEEP Agent and TAM can provide end-to-end security without relying on transport security.